addons/point_of_sale/static/src/js/models.js

```diff
@@ -2654,12 +2656,11 @@ export class Payment extends PosModel {
     }
     //exports as JSON for receipt printing
     export_for_printing() {
-        const ticket = escape(this.ticket).replace(/\n/g, "<br />"); // formatting
         return {
             cid: this.cid,
             amount: this.get_amount(),
             name: this.name,
-            ticket: Markup(ticket),
+            ticket: escape(this.ticket),
```

Comment:
After this change, the ticket looks like the following:

`ticket = "asdf: gasdf\ngadsf: 124123\n2351235: gawevnjg-jewfj-jkaasdf4iupowe-ghasf"`

<img width="356" alt="Screenshot 2023-04-17 at 16 53 17" src="https://user-images.githubusercontent.com/3245568/232524194-2b937212-ffa0-4e2d-9f63-f3d0119893d2.png">

Originally, it looks like this:

<img width="349" alt="Screenshot 2023-04-17 at 16 51 49" src="https://user-images.githubusercontent.com/3245568/232524400-8e67be78-88e9-426f-bfe7-aaf11a2acc1a.png">